app/views/stories/show.html.erb

```diff
@@ -76,3 +76,12 @@
     <% end %>
   </ol>
 <% end %>
+
```

Comment:
Please wrap the whole thing in `<% if @similar.any? %>` rather than always printing the `h4`.